exodus/exodus/core/apk.py

```diff
@@ -13,7 +13,6 @@
 from .celery import app
 from .static_analysis import download_apk, is_paid_app, clear_analysis_files, StaticAnalysis
 
-EXIT_CODE = -1
```

Comment:
**suggestion**: Better use constants rather than "magic numbers", even if the constants are only used once.